pyIGTLink/pyIGTLink.py

```diff
@@ -180,26 +217,47 @@ def __init__(self):
         self._timestamp = time.time() * 1000
 
         self._endian = ">"  # big-endian
+        self.crc = None
 
         self._binary_body = None
         self._binary_head = None
         self._body_pack_size = 0
+        self._body_size = 0
 
     def pack(self):
         binary_body = self.get_binary_body()
-        body_size = self.get_body_pack_size()
-        crc = CRC64(binary_body)
+        self._body_size = self.get_body_pack_size()
+        self.crc = CRC64(binary_body)
```

Comment:
actually, having these body size and arc inside the class (as self.crc and self.body_size) is not necessary. But i actually don't know if -- when we want to receive something -- we want to just return everything that was contained into the message in a dictionary (for example like line 262 of this file) or we want to modify the instance of MessageTransform, MessageBase or MessageImage to contain the actual message being received. For what concerns me we can also just return the info (which is easier) and keep changes to the members of the classes as small as possible.